### PR TITLE
dist: Support FreeBSD

### PR DESCRIPTION
**Pull request by jbergstroem**
Add setup scripts for two builders behind a vpn at Voxer. The configuration assumes you're deploying in a FreeBSD jail; we look up the default routed ip-adress through ansible and pass it to the init script.

/R=@rvagg

### CODE REVIEW

File: setup/freebsd/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# io.js Build FreeBSD Setup
+
+The current FreeBSD lives behind a VPN at [Voxer](http://voxer.com).
+If you want access, speak to [rvagg][rvagg] for further information.
```
Comment (rvagg) [inline setup/freebsd/README.md]: s/want/need, don't want to give Voxer the impression we're overly liberal about handing out access

Comment (jbergstroem) [inline setup/freebsd/README.md]: Ok, I'll elaborate.

File: setup/freebsd/resources/jenkins [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+jenkins_jnlpurl="https://jenkins-iojs.nodesource.com/computer/iojs-voxer-freebsd101-{{id}}/slave-agent.jnlp"
+jenkins_secret="{{secret}}"
+jenkins_jar="/home/iojs/slave.jar"
+jenkins_log_file="/home/iojs/jenkins_console.log"
+jenkins_args="-jnlpUrl ${jenkins_jnlpurl} -secret ${jenkins_secret}"
+jenkins_user="iojs"
+jenkins_group="iojs"
+
+pidfile="/var/run/jenkins/jenkins.pid"
```
Comment (rvagg) [inline setup/freebsd/resources/jenkins]: does the jenkins user need access to this file?

**Comment by rvagg** [comment]
lgtm

File: setup/freebsd/resources/jenkins (hunk 2 above)
Comment (jbergstroem) [inline setup/freebsd/resources/jenkins]: It's good practise to own the pidfile with the same user as the daemon. In this case it's not needed but I tried to stay close to [freebsd's own init script](https://svnweb.freebsd.org/ports/head/devel/jenkins/files/jenkins.in?revision=368098&view=markup).

**Comment by jbergstroem** [comment]
Thanks for the feedback. Merged in d8c0f1d4df143dac2e3cea05fe2e936eb3e75d00. Closing.

**Pull request closed without merging**